internal/sdkgen/sdkgen.go

```diff
@@ -43,6 +44,19 @@ func Generate(ctx context.Context, customerID, lang, schemaPath, outDir, baseURL
 		return fmt.Errorf("failed to read schema file %s: %w", schemaPath, err)
 	}
 
+	if !autoYes {
+		fmt.Print(fmt.Sprintf("Contents of %s will be overwritten. Continue? [Y/n]:", outDir))
```

Comment:
if I'm not mistaken, this should be `[y/N]`, right? Since hitting `return` will cause this to abort?

Reply:
yeah, but to be fair, I think this is the behavior we want, it's just a bit confusing